src/objects/conf/base.py

```diff
@@ -366,6 +366,20 @@
     "REMOTE_ADDR",
 )
 
+#
+# Sending EMAIL
+#
+EMAIL_HOST = config("EMAIL_HOST", default="localhost")
+EMAIL_PORT = config(
+    "EMAIL_PORT", default=25
+)  # disabled on Google Cloud, use 487 instead
+EMAIL_HOST_USER = config("EMAIL_HOST_USER", default="")
+EMAIL_HOST_PASSWORD = config("EMAIL_HOST_PASSWORD", default="")
+EMAIL_USE_TLS = config("EMAIL_USE_TLS", default=False)
+EMAIL_TIMEOUT = 10
+
+DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL", "objects@example.com")
```

Comment:
Note that DEFAULT_FROM_EMAIL and EMAIL_TIMEOUT are configured earlier (lines 208-209), these should be removed to avoid confusion